src/main/java/com/netflix/imflibrary/st2067_2/Application2E2021.java

```diff
@@ -0,0 +1,338 @@
+            this.colorModels = new HashSet<ColorModel>(colorModels);
+        }
+
+        public boolean has(Integer width,
+                Integer height,
+                Colorimetry colorSystem,
+                Integer bitDepth,
+                FrameLayoutType frameStructure,
+                Fraction frameRate,
+                Sampling sampling,
+                Quantization quantization,
+                ColorModel colorModel) {
+            return width <= this.maxWidth &&
+                    height <= this.maxHeight &&
+                    this.colorSystems.contains(colorSystem) &&
+                    this.bitDepths.contains(bitDepth) &&
+                    this.frameStructures.contains(frameStructure) &&
+                    this.frameRates.contains(frameRate) &&
+                    this.samplings.contains(sampling) &&
+                    this.quantizations.contains(quantization) &&
+                    this.colorModels.contains(colorModel);
+        }
+    }
+
+    static final Fraction[] FPS_HD = {
+        new Fraction(25),
+        new Fraction(30),
+        new Fraction(30000, 1001)
+    };
+
+    static final Fraction[] FPS_UHD = {
+        new Fraction(24),
+        new Fraction(24000, 1001),
+        new Fraction(25),
+        new Fraction(30),
+        new Fraction(30000, 1001),
+        new Fraction(50),
+        new Fraction(60),
+        new Fraction(60000, 1001)
+    };
+
+    static final Fraction[] FPS_4K = {
+        new Fraction(24),
+        new Fraction(24000, 1001),
+        new Fraction(25),
+        new Fraction(30),
+        new Fraction(30000, 1001),
+        new Fraction(50),
+        new Fraction(60),
+        new Fraction(60000, 1001),
+        new Fraction(120)
+    };
+
+    /* Table 3 at SMPTE ST 2067-21:2023 */
+    static final CharacteristicsSet[] IMAGE_CHARACTERISTICS = {
+        new CharacteristicsSet(
+            1920,
+            1080,
+            Arrays.asList(Colorimetry.Color1, Colorimetry.Color2, Colorimetry.Color3),
+            Arrays.asList(8, 10),
+            Arrays.asList(FrameLayoutType.SeparateFields),
+            Arrays.asList(FPS_HD),
+            Arrays.asList(Sampling.Sampling422),
+            Arrays.asList(Quantization.QE1),
+            Arrays.asList(ColorModel.YUV)
+        ),
+        new CharacteristicsSet(
+            1920,
+            1080,
+            Arrays.asList(Colorimetry.Color1, Colorimetry.Color2, Colorimetry.Color3),
+            Arrays.asList(8, 10),
+            Arrays.asList(FrameLayoutType.FullFrame),
+            Arrays.asList(FPS_HD),
+            Arrays.asList(Sampling.Sampling422),
+            Arrays.asList(Quantization.QE1),
+            Arrays.asList(ColorModel.YUV)
+        ),
+        new CharacteristicsSet(
+            1920,
+            1080,
+            Arrays.asList(Colorimetry.Color1, Colorimetry.Color2, Colorimetry.Color3),
+            Arrays.asList(8, 10, 12),
```

Comment:
No 12bit support for Color1 and Color2.
```suggestion
            Arrays.asList(8, 10),
```